scarb/tests/e2e/subcommand.rs

```diff
@@ -40,6 +40,78 @@ fn subcommand() {
         .stdout_eq("Hello beautiful world\n");
 }
 
+#[test]
+#[cfg_attr(
+    not(target_family = "unix"),
+    ignore = "This test should write a Rust code, because currently it only assumes Unix."
+)]
+fn list_commands_builtin_only() {
```

Comment:
these tests are going to be painful to maintain I think... each new command/slight change in docs would require to change this test

Wdyt about taking an unit-test-like approach and simply verify that `list-commands` includes hello, on CommandInfo-level? without ever looking into builtins